client/src/main/java/org/apache/rocketmq/client/consumer/MQPushConsumer.java

```diff
@@ -60,6 +60,7 @@
      * @param fullClassName full class name,must extend org.apache.rocketmq.common.filter. MessageFilter
      * @param filterClassSource class source code,used UTF-8 file encoding,must be responsible for your code safety
      */
+    @Deprecated
```

Comment:
Please describe the reason of why this API should deprecated, and list the version which will remove this API.